lib/mongo.ex

```diff
@@ -917,15 +922,6 @@ defmodule Mongo do
       opts: opts}
   end
 
-  defp singly_cursor(conn, coll, query, select, opts) do
```

Comment:
Thank you for deleting this, if only because it has a stupid name.

Reply:
It was throwing a no-used compilation warning